app/uat_actions/uat_actions/generate_tag_layout_template.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+# Will construct a tag layout template from existing tag groups
+class UatActions::GenerateTagLayoutTemplate < UatActions
+  DIRECTIONS = {
+    'InColumns (A1,B1,C1...)': 'TagLayout::InColumns',
+    'InRows (A1,A2,A3...)': 'TagLayout::InRows',
+    'InInverseColumns (H12,G12,F12...)': 'TagLayout::InInverseColumns',
+    'InInverseRows (H12,H11,H10...)': 'TagLayout::InInverseRows'
+  }.freeze
```

Comment:
Did you consider referencing the constant in the TagLayoutTemplatesController class, to avoid duplicating it?

Reply:
Tried again and got it working now, so that's a small improvement.